Realm/Realm.Weaver/Analytics/Analytics.cs

```diff
@@ -239,7 +250,7 @@ internal void AnalyzeUserAssembly(ModuleDefinition module)
                 _realmEnvMetrics[UserEnvironment.NetFramework] = _config.TargetFramework;
                 _realmEnvMetrics[UserEnvironment.NetFrameworkVersion] = _config.TargetFrameworkVersion;
 
-                foreach (var type in module.Types)
+                foreach (var type in module.Types.ToArray())
```

Comment:
Why is this necessary?

Reply:
This is executing while the weaver is running, which means that if the weaver adds new types (and it does for the schema), iteration will break.

Reply:
Makes sense. Thank you.